index.html

```diff
@@ -1,38 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<style>
-table,td,th{
-  border: 1px solid black;
-  border-collapse: collapse;
-   background-color: black ; 
-   opacity: 0.75;
```

Comment:
The lab asked for alpha transparency on the background colour, not transparency on the table element as a whole